tests/util/bft.py

```diff
@@ -212,13 +232,28 @@ def stop_replica(self, replica):
         Otherwise raise an AlreadyStoppedError.
         """
         if replica not in self.procs:
-            raise AlreadyStoppedError
+            raise AlreadyStoppedError(replica)
         p = self.procs[replica]
         p.kill()
         p.wait()
 
         del self.procs[replica]
 
+    def force_quorum_including_replica(self, replica_id, primary=0):
```

Comment:
Can you please add a doc comment here explaining what is going on?

Reply:
Fixed.